tests/chains/test_register.py

```diff
@@ -58,7 +56,7 @@ async def test_register_outgoing_worker(monkeypatch):
 
 @pytest.mark.asyncio
 async def test_outgoing():
-    assert len(OUTGOING_WORKERS.keys()) == 3  # 3 verifiers are included by default
+    assert len(OUTGOING_WORKERS) == 2  # 2 verifiers are included by default
```

Comment:
Do you know why this went from 3 to 2 ?

Reply:
The support for BNB was dropped, it was one of the 3.